solidity-frontend/src/main/java/org/sonarsource/solidity/frontend/Utils.java

```diff
@@ -51,11 +51,13 @@ public static boolean isCommentSignificant(String token) {
 
   public static String returnFunctionSignature(FunctionDefinitionContext ctx) {
     StringBuilder signature = new StringBuilder();
-    signature.append(ctx.identifier().getText());
-    ctx.parameterList().parameter().forEach(param -> signature.append(", " + param.typeName().getText()));
-    if (ctx.returnParameters() != null) {
-      ctx.returnParameters().parameterList().parameter().forEach(param -> signature.append(", " + param.typeName().getText()));
-    }
+    System.out.println("AAAAAAAAAAAAAAAAAAAAAAAa" + ctx);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/stylianos-agapiou-sonarsource-gi/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=org.sonarsource.solidity%3Asonar-solidity&pullRequest=39&issues=AWSE1m2Usk71k7lwfRFe&open=AWSE1m2Usk71k7lwfRFe)

[ISSUE_KEY]: AWSE1m2Usk71k7lwfRFe